src/easylink/configuration.py

```diff
@@ -279,16 +278,16 @@ def load_params_from_specification(
 
     This gathers the pipeline, input data, and computing environment specifications
     as well as the results directory into a single dictionary for insertion into
-    the ``Config`` object.
+    the :class:`Config` object.
 
     Parameters
     ----------
     pipeline_specification
-        The path to the pipeline specification yaml file.
+        The path to the pipeline specification file.
```

Comment:
Do we say somewhere else what these specifications files are supposed to look like?

Reply:
Not yet, no. I'd imagine that will show up in a concept or the quickstart or soemthing at some point. I just didn't feel the need to specify yaml here in case it ever does change or we support different formats